tests/PHPUnit/Framework/TestCase/SystemTestCase.php

```diff
@@ -58,6 +58,19 @@ abstract class SystemTestCase extends TestCase
      */
     public static $fixture;
 
+    public function setGroups(array $groups): void
```

Comment:
it's officially an `internal` method but seemed for me easiest way to make this work. Not sure there's another way.